### PR TITLE
enhancement(highlight): adding code to highlight lines in lessons

### PR DESCRIPTION
**Pull request by grantlouisherman**

### CODE REVIEW

File: src/components/CodeEditor.js [hunk 1]
```diff
@@ -169,11 +195,22 @@ class CodeEditor extends Component {
...
+      const { slides } = lesson
+      slide = slides[nextProps.tutorial.slideIndex]
+    }
     if (nextProps.game === 'SCRIPT-8 NEW') {
       this.setContents(blank)
     }
     if (nextProps.game.startsWith('SCRIPT-8 LESSON')) {
       this.setContents(nextProps.game.replace('SCRIPT-8 LESSON', ''))
+
+      if(slide && slide.lineToFoucs){
```
Comment (Kethku) [inline src/components/CodeEditor.js]: Should lineToFoucs be lineToFocus?

File: src/utils/lessons.yaml [hunk 2]
```diff
@@ -47,6 +47,7 @@
         print(42, 60, 'Hello world!', 0)
       }
     screen: CODE
+    lineToFoucs: 1
```
Comment (Kethku) [inline src/utils/lessons.yaml]: Same as above. I think this should be lineToFocus.

File: src/utils/lessons.yaml [hunk 3]
```diff
@@ -96,6 +97,7 @@
       draw = () => {
         print(42, 60, 'Hello world!', 0)
       }
+    lineToFoucs: 1
```
Comment (Kethku) [inline src/utils/lessons.yaml]: Dito here.

File: src/utils/lessons.yaml [hunk 4]
```diff
@@ -186,6 +188,7 @@
     - OK. We have reached the end. I hope you learned something new.
     - The next lesson will be about SCRIPT-8's magical number slider.
     - Are you curious? I would be. See you in the next one!
+  lineToFoucs: 1
```
Comment (Kethku) [inline src/utils/lessons.yaml]: Dito here.

**Comment by gabrielflorit** [comment]
Thank you for this. Will take a look asap.

File: src/components/CodeEditor.js (hunk 1 above)
Comment (grantlouisherman) [reply src/components/CodeEditor.js]: Ugh😭😭! Sorry about that, I will update when I get back my computer

**Comment by grantlouisherman** [comment]
hey @gabrielflorit @Kethku I updated, please take a look when you get a chance.

**Comment by gabrielflorit** [comment]
Thank you @grantlouisherman, this is great.

**Pull request merged**